six/modules/python/six/tests/runTests.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
I’ve fixed the tests, to a certain extent. Test_sicd_schemata fails instead of segfaults,  but it has a number of problems that will become much more sane to work with once the equality operators get merged in.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Sunday, April 10, 2016 11:05 AM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>
Subject: Re: [ngageoint/six-library] Regression testing (#57)


In six/modules/python/six/tests/runTests.py<http://cp.mcafee.com/d/1jWVIi41Aq43qb3BSrTs7c6XCSm6hMUQsLLCTNPb3a9EVd79J6XRTD3hPsQsL6QkmkkTPhOUM-r4qMSpfosH-ME2BkdrFOVJl3mWsKrju8erss_R-vov7fELZuVt5VctOab1EVKCO_ORQr8EGTupVkffGhBrwqrhdFCXYyUqenAkmkXL6XCOsVHkiP2cFASO7bV0zF7WOxfai3OVYy4tymNpIY_7NM68iFpKB9lwaNvd7abbXX0UT2Ivmfb0EzapdIxOVJyX31I5-Aq83iT0zF7WOxfai3OVYy4tymdbFEw1vAo_iAGMd44_Fek29Ewd506fzzhirvjdxVeM>:

> @@ -0,0 +1,56 @@

> +#!/usr/bin/env python

So is this my one-liner Python script I can run which will test everything?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/k-Kr6zqb3BSrTs7c6XCSm6hMUQsLLCTNPb3a9EVd79J6XRTD3hPsQsL6QkmkkTPhOUM-r4qMSpfosH-ME2BkdrFOVJl3mWsKrju8erss_R-vov7fELZuVt5VctOab1EVKCO_ORQr8EGTupVkffGhBrwqrjdFCXYyUqenAkmkXL6XCOsVHkiP2cFASO7bV0zF7WOxfai3OVYy4tymNpIY_7NSeOf-CnpIOYqenQQfpoJpBWXOpEVgKZV0QsI44hRQ7bvUUseg0ObVEVhpvvo76UlzWNVo54pj9JAendInoodwLQzh0qmU4t8_mk9VigunfAgzIiNFtd40bYz7WkBm1EwDZ9Owhd41EE0NYsqajrWpLQznkXIjh6I0>


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.